HW04.Rmd

```diff
@@ -29,33 +29,91 @@ Knitr is part of the knitr package. Allows you to change column names and add ca
 
 
 ```{r month, warning=FALSE}
-
+number_per_month <- gun_deaths %>% 
+    select(year, month) %>% 
+    group_by(year) %>%
+    count(month) #select and count number of deaths per month
+
+number_per_month$month <- factor(number_per_month$month) 
+levels(number_per_month$month) <- c("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec") #convert month to abbreviation
+    knitr::kable(x = number_per_month, format = "markdown", col.names = c("Year", "Month", "Number of Deaths"), align = "ccc", caption = "Number of Gun Deaths Per Month (2012-2014)")

```

Comment:
This use of knitr::kable() is nice! I loaded the knitr package in the beginning for the whole document. But since we really only needed the kable() function once, this is a great alternative.